homeassistant/components/foscam/camera.py

```diff
@@ -60,6 +144,18 @@ def __init__(self, device_info):
             if result == 0:
                 self._rtsp_port = response.get("rtspPort") or response.get("mediaPort")
 
+        _LOGGER.debug("RTSP port discovered: %s", self._rtsp_port)
```

Comment:
As above. If we really need the logging message it should be done outside of init method.